.github/ISSUE_TEMPLATE/release-check-list.md

```diff
@@ -10,27 +10,15 @@ assignees: ''
 
 ## Code freeze
 
-1. - [ ] Update image-rs to use the latest commit from ocicrypt-rs
-        
-    * https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml
-      * Change the revision
-      * Run `cargo update -p ocicrypt-rs`
-
-2. - [ ] Update image-rs to use the latest commit from attestation-agent
-
-    * https://github.com/confidential-containers/image-rs/blob/main/Cargo.toml
-      * Change the revision
-      * Run `cargo update -p attestation_agent`
-
-3. - [ ] Update Enclave CC to use the latest commit from image-rs
+- [ ] 1. Update Enclave CC to use the latest commit from image-rs
```

Comment:
Is this point of this step (vs. 11 below) to test compatibility with image-rs before tagging image-rs? I'm wondering if this can be removed so that we don't have "redo" steps below.

Reply:
This is to make sure that the version of image-rs that we are going to tag will actually work with enclave-cc. It's basically like having a release candidate. We follow this pattern with kata CCv0 as well.